nimbus-ui/nimbusui/src/app/components/platform/link.component.ts

```diff
@@ -134,6 +134,8 @@ export class Link extends BaseElement {
             } else if (this.element.leafState) {
                 this.url = this.resolveUrl();
             }
+            else
+            this.url = this.element.config.uiStyles.attributes.url;
```

Comment:
can this be added to the resolve url method where we do the necessary transformation based on leafstate ?